internal/rbd/group/volume_group.go

```diff
@@ -82,6 +85,12 @@ func GetVolumeGroup(
 		return nil, fmt.Errorf("failed to get volume attributes for id %q: %w", vg, err)
 	}
 
+	if attrs.GroupName == "" || attrs.CreationTime == nil {
```

Comment:
we could make use of `vg.GetName()` & `vg.GetCreationTime()` ?

Reply:
Right, we can do that.
Updated the code to use only `vg.GetName`